lib/cli.js

```diff
@@ -18,27 +18,41 @@ var getGitFolder = require('./getGitFolder');
 // hacky start if not run by mocha :-D
 // istanbul ignore next
 if (process.argv.join('').indexOf('mocha') === -1) {
+  var commitMsgFileOrText = process.argv[2] || getGitFolder() + '/COMMIT_EDITMSG';
 
-  var commitMsgFile = process.argv[2] || getGitFolder() + '/COMMIT_EDITMSG';
-  var incorrectLogFile = commitMsgFile.replace('COMMIT_EDITMSG', 'logs/incorrect-commit-msgs');
+  var bufferToString = function (buffer) {
```

Comment:
You could turn these anonymous function expressions to named function expressions. Usually anonymous functions are annoying during debugging 😄